dashboard/templates/common.html

```diff
@@ -10,8 +10,16 @@
 
 {% block stylesheets %}
 
-    <link href="{% static 'prototype/moj_template_overrides.css' %}" rel="stylesheet" type="text/css"/>
-    <link href="{% static 'prototype/main.css' %}" rel="stylesheet" type="text/css"/>
+    <link href="{% static 'prototype/styles/moj_template_overrides.css' %}" rel="stylesheet" type="text/css"/>
+    <link href="{% static 'prototype/styles/main.css' %}" rel="stylesheet" type="text/css"/>
+
+{% endblock %}
+
+{% block javascripts %}
+
+    <script src="https://cdn.plot.ly/plotly-1.10.2.min.js"></script>
+    <script src="https://code.jquery.com/jquery-2.2.3.js"></script>
```

Comment:
jquery is in general avoided in modern javascript. it's a bit like php in many ways. i can see that you are only using jquery for firing ajax calls. this is a better option https://github.com/visionmedia/superagent